content/influxdb/cloud-dedicated/admin/custom-partitions/view-partitions.md

```diff
@@ -108,40 +108,26 @@ with the name of the table you want to query information about.
 
 ---
 
-{{% code-placeholders "TABLE_NAME" %}}
+{{% code-placeholders "TABLE_NAME_(1|2|3)|TABLE_NAME" %}}
 
-### View partition templates of all tables
+### View the partition template of a specific table
 
 ```sql
-SELECT * FROM system.tables
```

Comment:
Removing the example for all the tables. Even though `SELECT * FROM system.tables` works fine, it can be expensive, so I don't want to encourage users to so it